### PR TITLE
GitHub flow

### PR DESCRIPTION
**Pull request by dariamine2000**

### CODE REVIEW

File: test.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (Hamzaoutdoors) [inline test.md]: - [ ] This file should be ignored and deleted remotely, Kindly delete it from GitHub as it should be hidden on Github

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @dariamine2000, I am **Hamza** your reviewer for this PR 😸 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

image

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README

## Required Changes ♻️
- [REQUIRED] Kindly add a description to this current Pull Request, it's a more professional resume about what you did and what you build in your future Pull Requests ^_^

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Good job creating a README file for your project, but I suggest you use [Microverse Template](https://github.com/microverseinc/readme-template)to have some more detailed description of your project 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @hamzaoutdoors in your question so I can receive the notification ^_^

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: test.md (hunk 1 above)
Comment (topeogunleye) [inline test.md]: - [ ] [REQUIRED] As the previous reviewer also stated. I see that you added test.md to your gitignore file but still, I can see it in your repo please fix that you can try removing test.md from your GitHub repo then try to push your local git repo again.

**Review by topeogunleye — CHANGES_REQUESTED** [review]
> # Changes Required
> ### Hi @dariamine2000,
> Nice job, you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Github flow is used :heavy_check_mark:.
> * You documented your work professionally :heavy_check_mark: .
> 
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * As the previous code reviewer did not mention this. Please add a more descritive PR title and description.
> See an example below:
>Screenshot from 2022-04-19 17-33-33

> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by topeogunleye — APPROVED** [review]
### Hi @dariamine2000, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
> 
> ## [Optional] suggestions:

- [ ] As I mentioned in the previous code review. Please kindly add a more descritive PR title and description.

> See an example below:
>Screenshot from 2022-04-19 17-33-33
 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**